hugegraph-core/src/main/java/org/apache/hugegraph/masterelection/StandardStateMachineCallback.java

```diff
@@ -94,12 +94,12 @@ public void error(StateMachineContext context, Throwable e) {
     public void initGlobalMasterInfo(StateMachineContext context) {
         StateMachineContext.MasterServerInfo master = context.master();
         if (master == null) {
-            this.globalMasterInfo.set(false, null);
+            this.globalMasterInfo.info(false, null);
             return;
         }
 
         boolean isMaster = Objects.equals(context.node(), master.node());
         String url = master.url();
-        this.globalMasterInfo.set(isMaster, url);
+        this.globalMasterInfo.info(isMaster, url);
```

Comment:
how frequently the initGlobalMasterInfo() to be called?

Reply:
frequently = 1 / heatbeat